File: programmers/js/level01/135808.js
// 과일 장수
function solution(k, m, score) {
  var answer = 0;

  score.sort((a, b) => b - a);

  for (let i = m; i <= score.length; i += m) {
    answer += score[i - 1] * m;
  }

  return answer;
}

File: programmers/java/level01/135808.java
// 과일 장수

import java.util.stream.IntStream;

class Solution {
    public int solution(int k, int m, int[] score) {
        int[] sortedScore = IntStream.of(score)
            .boxed()
            .sorted((a, b) -> b - a)
            .mapToInt(v -> v)
            .toArray();
        
        return IntStream.range(0, score.length)
            .filter(idx -> (idx + 1) % m == 0)
            .map(idx -> sortedScore[idx] * m)
            .sum();
    }
}